function/examples/internal/client/yaml_client.go

```diff
@@ -144,3 +145,7 @@ func (c *MapClient) List(ctx context.Context, opts metav1.ListOptions) (*unstruc
 	}
 	return &out, nil
 }
+
+func (c *MapClient) Watch(ctx context.Context, opts metav1.ListOptions) (watch.Interface, error) {
+	panic("not supported")
```

Comment:
In other methods, we used "not implemented". It's nothing big, but for consistency reasons I'd stick to one message :)

```suggestion
	panic("not implemented")
```